WalletWasabi.SDK/WalletWasabi.Gui/Global.cs

```diff
@@ -297,18 +306,7 @@ public async Task InitializeNoWalletAsync()
 				}
 				else
 				{
-					if (Config.UseTor)
-					{
-						// onlyForOnionHosts: false - Connect to clearnet IPs through Tor, too.
-						connectionParameters.TemplateBehaviors.Add(new SocksSettingsBehavior(Config.TorSocks5EndPoint, onlyForOnionHosts: false, networkCredential: null, streamIsolation: true));
-						// allowOnlyTorEndpoints: true - Connect only to onions and do not connect to clearnet IPs at all.
-						// This of course makes the first setting unnecessary, but it's better if that's around, in case someone wants to tinker here.
-						connectionParameters.EndpointConnector = new DefaultEndpointConnector(allowOnlyTorEndpoints: Network == Network.Main);
-
-						await AddKnownBitcoinFullNodeAsHiddenServiceAsync(AddressManager).ConfigureAwait(false);
-					}
-					Nodes = new NodesGroup(Network, connectionParameters, requirements: Constants.NodeRequirements);
-					Nodes.MaximumNodeConnection = 12;
+					Nodes = CreateAndConfigureNodesGroup(connectionParameters);
```

Comment:
All of the changes in WalletWasabi.Gui/Global.cs need to be applied to Chaincase.Common/Global.cs